datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/policy/ListPoliciesResolver.java

```diff
@@ -40,9 +46,16 @@ public CompletableFuture<ListPoliciesResult> get(final DataFetchingEnvironment e
       final Integer start = input.getStart() == null ? DEFAULT_START : input.getStart();
       final Integer count = input.getCount() == null ? DEFAULT_COUNT : input.getCount();
       final String query = input.getQuery() == null ? DEFAULT_QUERY : input.getQuery();
+      final List<FacetFilterInput> filters =
+          input.getFilters() == null ? Collections.emptyList() : input.getFilters();
+
+      log.info(
```

Comment:
Probably should be debug, might be fine as long as this resolver isn't called during normal execution flows.